### PR TITLE
feat:  iconpark official icon migration（a part of）

### PR DESCRIPTION
**Pull request by wuhuiluo**
## What this PR does / why we need it:
iconpark official icon migration

## Does this PR introduce a user interface change?

❎ No

## ChangeLog

| Language | Changelog |
| --------- | ------------ |
| 🇺🇸 English |              |
| 🇨🇳 中文    |              |

## Does this PR need be patched to older version?
❎ No

## Which issue(s) this PR fixes:

Fixes #

### CODE REVIEW

File: shell/app/common/components/contractive-filter/index.scss [hunk 1]
```diff
@@ -16,10 +16,18 @@
...
+    color: $lightgray;
+  }
+
   &:hover {
     .contractive-filter-item-close {
       display: initial;
     }
+
+    .hover {
+      color: gray;
```
Comment (Zero-Rock) [inline shell/app/common/components/contractive-filter/index.scss:29]: use scss variable

File: shell/app/common/components/pure-log-roller/log-roller.tsx [hunk 2]
```diff
@@ -103,8 +102,8 @@ export class LogRoller extends React.Component<IProps, IState> {
     } = this.props;
     const { fullScreen } = this.state;
     let logContent = rolling ? (
-      <div>
-        Loading... <IconLoading spin />
+      <div className="flex">
+        Loading... <ErdaIcon className="ml-1" type="loading" color="currentColor" spin />
```
Comment (Zero-Rock) [inline shell/app/common/components/pure-log-roller/log-roller.tsx:106]: CurrentColor should be the default props

File: shell/app/modules/cmp/pages/cluster-manage/cloud-machine-form-modal.tsx [hunk 3]
```diff
@@ -127,9 +126,13 @@ const CloudMachineAddForm = (props: any) => {
       <BasicForm form={form} />
       {['alicloud-cs', 'alicloud-cs-managed'].includes(cloudVendor) ? null : (
         <div className="more">
-          <a className="more-btn" onClick={() => setShowMore(!showMore)}>
+          <a className="more-btn w-20 flex" onClick={() => setShowMore(!showMore)}>
```
Comment (Zero-Rock) [inline shell/app/modules/cmp/pages/cluster-manage/cloud-machine-form-modal.tsx:129]: same as above

File: shell/app/modules/cmp/pages/cluster-manage/add-cluster-forms/cluster-form.tsx [hunk 4]
```diff
@@ -276,9 +275,13 @@ const ClusterAddForm = (props: any) => {
       />
       {clusterType === 'edas' ? null : (
         <div className="more">
-          <a className="more-btn" onClick={() => setShowMore(!showMore)}>
+          <a className="more-btn w-20 flex" onClick={() => setShowMore(!showMore)}>
```
Comment (Zero-Rock) [inline shell/app/modules/cmp/pages/cluster-manage/add-cluster-forms/cluster-form.tsx:278]: whether the width is large enough in en mode

File: shell/app/menus/dop.tsx [hunk 5]
```diff
@@ -33,13 +26,13 @@ export const getDopMenu = () => {
       [
         {
           href: goTo.resolve.dopRoot(), // '/dop/projects',
-          icon: <IconApiApp />,
+          icon: <ErdaIcon className="mt-3.5 mr-1" type="api-app" color="currentColor" />,
```
Comment (Zero-Rock) [inline shell/app/menus/dop.tsx:29]: `mt-3.5 mr-1`  cannot be set where they are used？contains all menu config

File: shell/app/common/components/contractive-filter/index.scss (hunk 1 above)
Comment (wuhuiluo) [reply shell/app/common/components/contractive-filter/index.scss]: modified

File: shell/app/common/components/pure-log-roller/log-roller.tsx (hunk 2 above)
Comment (wuhuiluo) [reply shell/app/common/components/pure-log-roller/log-roller.tsx]: modified

File: shell/app/modules/cmp/pages/cluster-manage/cloud-machine-form-modal.tsx (hunk 3 above)
Comment (wuhuiluo) [reply shell/app/modules/cmp/pages/cluster-manage/cloud-machine-form-modal.tsx]: modified

File: shell/app/modules/cmp/pages/cluster-manage/add-cluster-forms/cluster-form.tsx (hunk 4 above)
Comment (wuhuiluo) [reply shell/app/modules/cmp/pages/cluster-manage/add-cluster-forms/cluster-form.tsx]: modified

File: shell/app/menus/dop.tsx (hunk 5 above)
Comment (wuhuiluo) [reply shell/app/menus/dop.tsx]: modified

**Pull request merged**